src/data/tests/EnterpriseAppTestData/index.js

```diff
@@ -0,0 +1,6 @@
+import { EnterpriseAppContext, initialStateValue } from './context';
+
+export default {
```

Comment:
nit: If you're treating the default export as an object, you might use named exports instead with something like:

```jsx
export { initialStateValue, EnterpriseAppContext } from './context';
```